packages/nimble-components/src/patterns/anchor/tests/anchor-patterns.stories.ts

```diff
@@ -0,0 +1,157 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { html, ref } from '@microsoft/fast-element';
+import { createUserSelectedThemeStory } from '../../../utilities/tests/storybook';
+import {
+    controlLabelFont,
+    controlLabelFontColor,
+    mediumPadding,
+    standardPadding
+} from '../../../theme-provider/design-tokens';
+import { anchorTag } from '../../../anchor';
+import { anchorButtonTag } from '../../../anchor-button';
+import { anchorTabsTag } from '../../../anchor-tabs';
+import { anchorTabTag } from '../../../anchor-tab';
+import { breadcrumbTag } from '../../../breadcrumb';
+import { breadcrumbItemTag } from '../../../breadcrumb-item';
+import { RichTextViewer, richTextViewerTag } from '../../../rich-text/viewer';
+import { anchorTreeItemTag } from '../../../anchor-tree-item';
+import { treeViewTag } from '../../../tree-view';
+import { anchorMenuItemTag } from '../../../anchor-menu-item';
+import { menuTag } from '../../../menu';
+import { Table, tableTag } from '../../../table';
+import { tableColumnAnchorTag } from '../../../table-column/anchor';
+
+const hrefDescription = 'To disable the control, remove the `href` attribute.';
+
+interface AnchorPatternsArgs {
+    label: string;
+    href: string;
+    tableRef: Table;
+    setTableData: (args: AnchorPatternsArgs) => void;
+    richTextViewerRef: RichTextViewer;
+    setRichTextViewerData: (args: AnchorPatternsArgs) => void;
+}
+
+const metadata: Meta<AnchorPatternsArgs> = {
+    title: 'Tests/Anchor Patterns',
+    parameters: {
+        actions: {}
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        <style class='code-hide'>
+            .control-container {
+                margin: var(${standardPadding.cssCustomProperty});
+            }
+
+            .label {
+                font: var(${controlLabelFont.cssCustomProperty});
+                color: var(${controlLabelFontColor.cssCustomProperty});
+                margin-bottom: var(${mediumPadding.cssCustomProperty});
+            }
+        </style>
+        <div class="control-container">
+            <div class="label">Native anchor</div>
+            <a href="${x => x.href}">${x => x.label}</a>
```

Comment:
Could we make it so that each url is different with a query parameter, something like `https://nimble.ni.dev?type=native-anchor`

```suggestion
            <a href="${x => x.disabled ? '' : 'https://nimble.ni.dev?type=native-anchor'}">${x => x.label}</a>
```

It would make it a lot easier tabbing between them to see the browser native url preview thing update.